internal/controller/eventing/integrationtests/validation/integration_test.go

```diff
@@ -0,0 +1,683 @@
+package validation_test
```

Comment:
Does it make sense to move this file beside the functionality being tested?

- internal/controller/eventing/controller.go
- internal/controller/eventing/controller_test.go

Reply:
I am following the structure like it is done in the upcoming PR [here](https://github.com/kyma-project/eventing-manager/blob/bb7b8867cac3987bac279f5103739f19988f4b8e/internal/controller/eventing/integrationtests/controller/integration_test.go#L1) and how it is already done in the nats-manager [here](https://github.com/kyma-project/nats-manager/blob/46d9dda8ecc2677272c78401940ecc57bba0ca2f/internal/controller/nats/integrationtests/validation/integration_test.go#L1). 

So I would keep it like this unless there are strong reasons to move it beside the controller?